ioctl/cmd/account/accountcreate.go

```diff
@@ -61,15 +61,28 @@ type generatedAccount struct {
 }
 
 func init() {
-	accountCreateCmd.Flags().UintVarP(&numAccounts, "num", "n", 1, config.TranslateInLang(flagNumUsages, config.UILanguage))
+	accountCreateCmd.Flags().UintVarP(&numAccounts, "num", "n", 1,
+		config.TranslateInLang(flagNumUsages, config.UILanguage))
+	accountCreateCmd.Flags().BoolVar(&CryptoSm2, "sm2", false,
+		config.TranslateInLang(flagSm2Usage, config.UILanguage))
```

Comment:
why we need a flag?

Reply:
> why we need a flag?

I make ioctl support both two cryptography algorithms. This flag is to swtich them when using command like `account create`, avoiding more commands for similar functions.

Reply:
ioctl supports both two cryptography algorithms. xctl only supports sm2. How I implement xctl is to set `CryptoSm2` to `true` always.